cinema/permitions.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS

```

Comment:
There is a syntax error in the import statement. You should use a comma `,` instead of a slash `/` to separate imported modules: `from rest_framework.permissions import BasePermission, SAFE_METHODS`.